_includes/releases/v23.1/v23.1.0-alpha.3.md

```diff
@@ -0,0 +1,103 @@
+
+<h3 id="v23-1-0-alpha-3-db-console-changes">DB Console changes</h3>
+
+- Added execution insights to the [Statement Fingerprint Details](https://www.cockroachlabs.com/docs/cockroachcloud/statements-page.html#sql-statement-fingerprints) and [Transaction Fingerprint Details](https://www.cockroachlabs.com/docs/stable/ui-transactions-page.html#transaction-fingerprints-view) Pages. [#96440][#96440]
+- Txn insights that were not found will now display a message `Insight not found`. [#96832][#96832]
+- Added waiting statement ID and fingerprint to the [insights transaction](https://www.cockroachlabs.com/docs/stable/ui-insights-page.html#transaction-executions-view) details page. Added blocking transaction ID and fingerprint to the the [insights statement](https://www.cockroachlabs.com/docs/stable/ui-insights-page.html#statement-executions-view) page. [#96872][#96872]
+
+<h3 id="v23-1-0-alpha-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where casting a [TimeTZ](https://www.cockroachlabs.com/docs/stable/time.html) to an array results in displaying second offsets even if they are zero. [#96583][#96583]
+- Allowed [`ALTER TABLE .. ADD/DROP CONSTRAINT .. NOT VALID`](https://www.cockroachlabs.com/docs/stable/alter-table.html) and [`VALIDATE CONSTRAINT ..`](https://www.cockroachlabs.com/docs/stable/alter-table.html#validate-constraints) to behave consistently with PostgreSQL. Previously, the `VALIDATE CONSTRAINT` would fail and cause the whole statement to fail.[#96648][#96648]
+- Resolved the [TimestampTZ](https://www.cockroachlabs.com/docs/stable/timestamp.html) to match PostgreSQL. We previously included the minute/second offset for TimestampTZ in certain places when casting it to string even when they were zero. [#96833][#96833]
+- Resolved using negative years instead of BC when casting a [TimestampTZ](https://www.cockroachlabs.com/docs/stable/timestamp.html) to a string. [#96833][#96833]
+- Fixed the [`SHOW GRANTS FOR public`](https://www.cockroachlabs.com/docs/stable/show-grants.html) command so it works correctly. Previously, this would return an error saying that the `public` role does not exist. [#96957][#96957]
+- Statement source (square bracket) syntax is no longer allowed in [user-defined functions](https://www.cockroachlabs.com/docs/stable/user-defined-functions.html). Prior to this fix, using this syntax in a UDF would cause a panic. This restriction will be lifted in the future. [#96824][#96824]
+
+<h3 id="v23-1-0-alpha-3-performance-improvements">Performance improvements</h3>
+
+- The execution of multiple [FOREIGN KEY](https://www.cockroachlabs.com/docs/stable/foreign-key.html) and [UNIQUE](https://www.cockroachlabs.com/docs/stable/unique.html) constraint checks have been parallelized in some cases. As a result, these checks should be completed faster, especially so in multi-region environments where the checks require cross-region reads. [#96123][#96123]
```

Comment:
... particularly in multi-region environments... 

Foreign key and unique are both SQL statements here, so they need backticks.